### ISSUE TITLE
Decide If The Travel Fund Should Cover Visa Expenses

### ISSUE THREAD

**Issue by bensternthal** [openjs-foundation/cross-project-council#1240]
This was surfaced in the initial review of the travel fund work. 

Should the travel fund cover visa costs?

**Comment by ljharb** [comment]
I would assume that a travel fund covers all non-discretionary travel-related spending, which definitely includes visa costs.

**Comment by RaisinTen** [comment]
@bensternthal this issue has been open for over a year now. Has there been any progress on this? What needs to be done to move this forward?

Not funding visas is a major deterrent to the participation of underrepresented groups of people in the collab summits and actively harms diversity.

Regardless of being a Node.js TSC member and an active core contributor since 2020, I will not able to attend the Node.js collab summits in-person just because I'm from India and I need a visa to go to any country where the collab summits are hosted.

**Comment by Qard** [comment]
I agree with @ljharb that anything which can be reasonably expected to be a requirement for one to get to and attend the events should qualify for reimbursement. Needing a visa is not an uncommon thing, especially when events have been held in US. I would have just assumed this was already included.

**Comment by mcollina** [comment]
The reason why Visa costs are not refundable is that it's an unbound cost with no guarantee of success. Moreover, those are often reusable and not specific for a single trip. Last but not least, we refund _after_ the trip: if a visa is denied, should we refund it anyway?

**Comment by RaisinTen** [comment]
> The reason why Visa costs are not refundable is that it's an unbound cost with no guarantee of success. Moreover, those are often reusable and not specific for a single trip.

Visa costs used to be refunded previously from 2017 to 2023:

- nodejs/TSC - https://github.com/nodejs/TSC/pulls?q=visa
- nodejs/admin - https://github.com/nodejs/admin/pulls?q=visa
- OpenJSF - https://github.com/openjs-foundation/community-fund/pull/28

Given that visa costs were refunded without issues for 6 years, it’s unclear why this is suddenly a problem now.

> Last but not least, we refund after the trip: if a visa is denied, should we refund it anyway?

I think that should be done, yes.

Visa costs, whether approved or denied, should not be a financial burden on a collaborator.

**Comment by RaisinTen** [comment]
I'm also cc'ing @eemeli @ruyadorno @tobie @WaleedAshraf in case you'd like to chime in here, as you've all interacted with this comment in the policy creation PR - https://github.com/openjs-foundation/cross-project-council/pull/1230#discussion_r1457126321

**Comment by mertcanaltin** [comment]
hello, I support people who are approved to come for collab to get a refund if their visa is rejected, so that their motivation is not broken.

**Comment by Qard** [comment]
> The reason why Visa costs are not refundable is that it's an unbound cost with no guarantee of success. Moreover, those are often reusable and not specific for a single trip.

Can we pre-approve visa reimbursements? As in the person files a request saying "I want to apply for reimbursement for visa X with Y and Z cost and requirements." then we approve reimbursement of the application for that specific component of the trip rather than putting that after like everything else? We could still _pay_ it after, just have a pre-approval process and we can say "Yes, you have $$$ to do this based on the related costs you have submitted for the application."

**Comment by eemeli** [comment]
I would consider visa costs the same way as other refundable travel expenses, i.e. with their own category in the application form. I would also consider having a visa denied the same way as other reasons why planned travel might not be possible. I'm not sure what that is, but I don't see any reason why visa expenses should get any special treatment.

**Comment by gireeshpunathil** [comment]
excluding visa expense reimbursement contradicts the principles of inclusivity and diversity in multiple ways. anyone who argues against its inclusion on technical grounds please be sensitive about how that can potentially become barrier to participation and skew representation.

**Comment by ruyadorno** [comment]
> excluding visa expense reimbursement contradicts the principles of inclusivity and diversity in multiple ways. anyone who argues against its inclusion on technical grounds please be sensitive about how that can potentially become barrier to participation and skew representation.

Agree 100%

> The reason why Visa costs are not refundable is that it's an unbound cost with no guarantee of success. Moreover, those are often reusable and not specific for a single trip. Last but not least, we refund after the trip: if a visa is denied, should we refund it anyway?

Those are some valid technical challenges but I don't agree the answer should be to not cover visa expenses at all. I see folks are already proposing solutions in this thread and I trust the CPC with the help of the community can design a framework that makes it possible to cover visa expenses.

**Comment by WaleedAshraf** [comment]
@RaisinTen, thanks for tagging.

I reimbursed the visa cost once (maybe twice) in 2019/2020 when I was part of a community committee. Otherwise, it would not have been possible for me to attend.

I think it should be covered, irrespective of whether the visa is approved or not.

**Comment by rginn** [comment]
I support adding visa fees as a refundable expense. Historically, we’ve followed the Linux Foundation Travel Fund policy, however I’ve confirmed with the LF that there is no legal restriction (or otherwise) that would prevent us from changing the OpenJS policy to allow for the reimbursement of travel visa fees.

**Comment by ctcpip** [comment]
policy itself resolved via #1452 but leaving open to follow up with "changes to the intake forms" and anything else

**Comment by bensternthal** [comment]
@bensternthal add visa item to intake form!

**Comment by bensternthal** [comment]
Intake form has been updated and tested.https://forms.gle/rEAWmoztQrCzfE8GA
